packages/components-native/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@jobber/components-native",
-  "version": "0.60.0",
```

Comment:
Not sure where these changes to components-native came from. I think we should up this change and the one on the change log. I assume a conflict was resolved incorrectly or something